.travis.yml

```diff
@@ -1,9 +1,25 @@
+dist: xenial
 language: python
 python:
   - "2.7"
+  - "3.6"
+
 install:
-  - pip install --upgrade pip
-  - pip install --upgrade numpy scipy matplotlib pandas
-  - pip install --upgrade scikit-learn
+  - pip install tox-travis
+
 script:
-  - python -m unittest discover -s test -p '*_test.py'
+  - tox
+
+
+# The following section deploys to pypi when a version tag is applied to master, see https://docs.travis-ci.com/user/deployment/pypi/
+deploy:
+  - provider: pypi
+    user: CBampis
+    password:
+      secure: "to be filled in"
+    distributions: "sdist bdist_wheel --universal"
```

Comment:
This would publish source tarball + a universal wheel (will have to double-check that a universal wheel is indeed possible..)